src/datatypes/array.h

```diff
@@ -49,6 +49,14 @@ u_int32_t SIArray_Length(SIValue siarray);
   */
 bool SIArray_ContainsType(SIValue siarray, SIType t);
 
+/**
+  * @brief  Returns true if the array contains an element equals to 'value'
+  * @param  siarray: array
+  * @param  value: value to search for
+  * @retval a boolean indicating whether value was found in siarray
+  */
+bool SIArray_ContainsValue(SIValue siarray, SIValue value);
```

Comment:
Update `SIValue AR_IN(SIValue *argv, int argc, void *private_data) {` `list_funcs.c` to use this new function.

Reply:
@swilly22 I believe other combinations are already supported and tested. Do you really think we should put in extra work now just to limit it?